### ISSUE TITLE
[Feature Request] Transparency settings for the dock/panel.

### ISSUE THREAD

**Issue by DuttaAB-dev** [pop-os/gnome-shell-extension-pop-cosmic#31]
By default there is no option to change the transparency settings for the dock so it would be great to be able to change that and also possibly the transparency of the top bar as default gnome does not allow that, which is quiet a bummer as many people opt to have a more translucent dock/top panel to go along with their styling. Also an option to make the dock change transparency dynamically just like the default behaviour of dash to dock would be a great addition!

**Comment by mimbrero** [comment]
Yes please XD

**Comment by uco-andres** [comment]
100% Agree

**Comment by TC68** [comment]
Yes please

**Comment by dabitCoder** [comment]
It would be very nice to customize this

**Comment by OldmanHades** [comment]
There isn't even a way to do this with dconf editor.  Please add in some sort of customization options.

**Comment by dev-andreas** [comment]
Would be great

**Comment by thisisapipitoworld** [comment]
Think about it. :+1:

**Comment by QNecron** [comment]
Color and opacity would be a great thing to be able to change, got to match my wallpapers after all! :rofl:

**Comment by Fjolserik** [comment]
I totally agree with the above
Color and opacity, in both dock and proces bar, would be a great thing to be able to change, got to match my wallpapers after all! 
It's freedom to be able to have a choice.

**Comment by ghost** [comment]
Yes please, me wait.

**Comment by Auxority** [comment]
Oi yes, I started using Pop Os today, and this is one of the features I'm missing. But I really love the OS so far :)

**Comment by youdontneedtoknow22** [comment]
I wonder why the devs seem to (almost) never respond to issues/requests. Just a simple (It's on our plan) or (we don't want this) is enough.

**Comment by DuttaAB-dev** [comment]
> I wonder why the devs seem to (almost) never respond to issues/requests. Just a simple (It's on our plan) or (we don't want this) is enough.

They used to, but dunno what happened. The situation is even worse in the discussions page.

**Comment by leviport** [comment]
@DuttaAB-dev We're still here, and we still see everything. We just don't always have much to say. This issue, for example, is pretty straightforward. We'd like to implement transparency setting for the dock, but it's not as high of a priority as a bunch of the other things we're currently working on. We're still tracking this issue for when we're able to work on it though.

**Comment by DuttaAB-dev** [comment]
> @DuttaAB-dev We're still here, and we still see everything. We just don't always have much to say. This issue, for example, is pretty straightforward. We'd like to implement transparency setting for the dock, but it's not as high of a priority as a bunch of the other things we're currently working on. We're still tracking this issue for when we're able to work on it though.

Sorry, I did not mean to sound that rude but I hope you understand that it is sometimes reassuring to get the issue acknowledged by the devs. Else it feels being neglected. 

Thank you.

**Comment by leviport** [comment]
No worries, no offense was taken. I understand your point, and I'll try to show my engagement more. Thanks for the feedback!

**Comment by leoriohub** [comment]
This would look sick!

**Comment by leoriohub** [comment]
This would look incredible! I wish it would be implemented :(

**Comment by krecke** [comment]
Totally agree, it would be great - mainly for the dock.

![image](https://user-images.githubusercontent.com/625173/149813734-fa8af09b-1df4-4ff2-a1f4-afdaa73c55cd.png)

**Comment by kvetoslavnovak** [comment]
Yeah, Transparency/background color settings for the dock would be great.

**Comment by andriolisp** [comment]
This will be awesome 💥💥

**Comment by leviport** [comment]
By the way, this can now be set in dconf-editor under `/org/gnome/shell/extensions/dash-to-dock/background-opacity`
![2022-02-17_06-25](https://user-images.githubusercontent.com/13512097/154491569-6a32b726-4c4a-4607-af7d-9b9bc25fa525.png)

**Comment by TC68** [comment]
> By the way, this can now be set in dconf-editor under `/org/gnome/shell/extensions/dash-to-dock/background-opacity` ![2022-02-17_06-25](https://user-images.githubusercontent.com/13512097/154491569-6a32b726-4c4a-4607-af7d-9b9bc25fa525.png)

Fantastic tip. Works perfectly for me but it would still be nice to include in the system settings

**Comment by lavininhamelo** [comment]
You can use this command on the Terminal to make the transparency:

`gsettings set org.gnome.shell.extensions.dash-to-dock background-opacity 0.1`

PS: Don't forget to change the opacity to your preference.

**Comment by leviport** [comment]
Yes, that's another way to change that setting.

**Comment by Itz-Abhishek-Tiwari** [comment]
Hey there! If you're looking for an easy way to customize your dock and application menu, this script might just be what you need!
https://github.com/Itz-Abhishek-Tiwari/pop-os-dock-application-color-changer
